### ISSUE TITLE
[fd3a94] Internal WCAG links are not resolved/brackets remain

### ISSUE THREAD

**Issue by joshuamaker8-create** [auto-wcag/auto-wcag#2347]
https://www.w3.org/WAI/standards-guidelines/act/rules/b20e66/proposed/

Under Background, this text has what I assume is supposed to be dynamically filled links, but they did not render/fill. The links seem correct in the page source, but is not rendering correctly.

Pages with links that are not [ambiguous to users in general][], but are ambiguous to some users are likely to fail [success criterion 1.3.1 Info and Relationships][sc131] if the disambiguation information is conveyed through presentation, but not semantically. Moreover, when this information is non-text content, such a page is likely to fail [success criterion 1.1.1 Non-text Content][sc111].

**Comment by joshuamaker8-create** [comment]
Added link to page where error was found.

**Comment by daniel-montalvo** [comment]
Thanks @joshuamaker8-create for bringing this to our attention. 

There is something we need to do to our source markdown files for these links to render, and probably we've forgotten about it. I'll take a pass at it and get back when I have news.

**Comment by daniel-montalvo** [comment]
@joshuamaker8-create 
There's a fix in progress for this.

Thanks for reporting
